app/controllers/api/v0x1/portfolio_items_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
         if params[:portfolio_id]
           collection(PortfolioItem.where(:portfolio_id => params.require(:portfolio_id)))
```

Comment:
@lindgrenj6 Do we need a default scope ( or facility provided by the `Discard` gem ) to make sure `where` will not return any discarded `PortfolioItem` record?

Same problem could exist [here](https://github.com/ManageIQ/service_portal-api/blob/master/app/controllers/api/v0x1/portfolios_controller.rb#L11-L12) as well.

Reply:
Had a discussion, updated the scope to exclude discarded items by default + added appropriate tests.